compiler/quilt/nodes.py

```diff
@@ -258,11 +256,12 @@ class PackageNode(GroupNode):
     """
     def __init__(self, package, meta):
         super(PackageNode, self).__init__(meta)
-        self._package = package
+        self._package = None
+        self._node = package if package is not None else None
```

Comment:
We really shouldn't store the `_node`. We're now back to keeping two package trees in memory.

Reply:
I agree, but this is just the first step. We used to store a reference to package object, which contained a reference to this root node. This just cuts out the Package object.

Reply:
I looked into removing it and it won't be easy until we merge core nodes and import nodes. The main issue is that the core node is the only way to get the package hash. I propose stopping here with this PR, then merging core and import nodes in a separate PR. Once we do that, we'll be down to a single tree.